templates/participants.html

```diff
@@ -68,3 +79,63 @@ <h1>Wszyscy uczestnicy</h1>
     </article>
 </div>
 {% endblock %}
+
+{% block script %}
+  <script>
+    $(document).ready(() => {
+      const table = $('#participants-table').DataTable({
```

Comment:
Now that we have an option to select columns and expand an entry, can we add other options from the "People info" tab here? (only pesel, address, t-shirt and comments - other are not even used anymore)